src/components/collection.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { Box } from '@mui/material';
+import { useUser } from '@auth0/nextjs-auth0/client';
+
+const Collection = () => {
+
+  const { user, error, isLoading } = useUser();
+
+  // Make a loading component later to make this more interesting
+  if (isLoading) return <div>Loading...</div>
+  if (error) return <div>{error.message}</div>
+
+  // If logged in then display a welcome message otherwise prompt user to login
+  const renderCollection = () => {
+    if (user) {
+      return <h2>This is where the persons collection will display</h2>
+    } else {
+      return <h2>Please login to your account</h2>
+    }
+  }
```

Comment:
I wouldn't let them get to this page without being logged in at all as I don't see a point

You could setup a redirect if there is no user present.